src/renderer/Extensions/ApplicationSearch/Linux/LinuxSettings.tsx

```diff
@@ -0,0 +1,28 @@
+import { useExtensionSetting } from "@Core/Hooks";
+import { Section } from "@Core/Settings/Section";
+import { SectionList } from "@Core/Settings/SectionList";
+import { Field, Input } from "@fluentui/react-components";
+
+export const LinuxSettings = () => {
+    const extensionId = "ApplicationSearch";
+
+    const { value: folders } = useExtensionSetting<string[]>({
+        extensionId,
+        key: "linuxFolders",
+    });
+
+    return (
+        <SectionList>
+            <Section>
+                <Field
+                    label="Application Folders (Hardcoded - Cannot be changed)"
+                    style={{ display: "flex", flexDirection: "column", gap: 5 }}
+                >
+                    {folders.map((folder) => (
+                        <Input key={`linuxFolder-${folder}`} value={folder} readOnly={true} disabled={true} />
+                    ))}
+                </Field>
```

Comment:
If the folders are hard-coded, I suggest not showing them in the settings. Looks like we don't need any settings for the ApplicationSearch on Linux at all. I would remove this.